tests/regtest/test_services_create_invoice.py

```diff
@@ -0,0 +1,32 @@
+import pytest
+from bolt11 import decode
+
+from lnbits.core.services import (
+    create_invoice,
+)
+
+descrtiption = "test create invoice"
+
+
+@pytest.mark.asyncio
+async def test_create_invoice(from_wallet):
+    payment_hash, pr = await create_invoice(
+        wallet_id=from_wallet.id,
+        amount=1000,
+        memo=descrtiption,
+    )
+    invoice = decode(pr)
+    assert invoice.payment_hash == payment_hash
+    assert invoice.amount_msat == 1000000
+    assert invoice.description == descrtiption
+
+
+@pytest.mark.asyncio
+async def test_create_internal_invoice(from_wallet):
+    payment_hash, pr = await create_invoice(
```

Comment:
`internal` uses `FakeWallet`
for the test to be complete and assertion should be made that the invoice is not present on the real funding source.